From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Xose Vazquez Perez <xose.vazquez@gmail.com>
Date: Sat, 20 Mar 2021 03:13:07 +0100
Subject: [PATCH] multipath-tools: use same format for default values in
 multipath.conf.5

For max_sectors_kb, replace "device dependent" with its sysfs path.
And use <dev> as wildcard for device in paths.

Cc: Martin Wilck <mwilck@suse.com>
Cc: Benjamin Marzinski <bmarzins@redhat.com>
Cc: Christophe Varoqui <christophe.varoqui@opensvc.com>
Cc: DM-DEVEL ML <dm-devel@redhat.com>
Reviewed-by: Martin Wilck <mwilck@suse.com>
Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 multipath/multipath.conf.5 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/multipath/multipath.conf.5 b/multipath/multipath.conf.5
index 2aa0f526..064e4826 100644
--- a/multipath/multipath.conf.5
+++ b/multipath/multipath.conf.5
@@ -647,7 +647,7 @@ does not respond to the checker command after \fIchecker_timeout\fR
 seconds have elapsed, it is considered down.
 .RS
 .TP
-The default is: in \fB/sys/block/sd<x>/device/timeout\fR
+The default is: in \fB/sys/block/<dev>/device/timeout\fR
 .RE
 .
 .
@@ -1242,7 +1242,7 @@ Sets the max_sectors_kb device parameter on all path devices and the multipath
 device to the specified value.
 .RS
 .TP
-The default is: \fB<device dependent>\fR
+The default is: in \fB/sys/block/<dev>/queue/max_sectors_kb\fR
 .RE
 .
 .
